### PR TITLE
9.0 add mrp sales info module from 8.0

### PR DESCRIPTION
**Pull request by victor-ditay**
cc @elicoidal

### CODE REVIEW

**Comment by elicoidal** [comment]
CLA solved: covered by Elico Corp CLA

File: mrp_sale_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+* Add fields to Manufacturing Orders and Work Orders:
+    * Sale order
+    * Customer
+    * Commitment Date
+
+Usage
+=====
+
+New fields are in Tree views and in Extra information tab at form views.
```
Comment (elicoidal) [inline mrp_sale_info/README.rst]: New fields are displayed in tree and in form views (Extra information tab).

File: mrp_sale_info/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+MRP Sale Info
+=============
+
+This module extends the functionality of *sale_mrp*.
```
Comment (elicoidal) [inline mrp_sale_info/README.rst]: This module extends the functionality of _sale_mrp_ and adds related fields to Manufacturing Orders and Work Orders:

**Comment by elicoidal** [comment]
some minor details. other than that, LGTM
@pedrobaeza ?

**Comment by pedrobaeza** [comment]
Apart from the comments from @elicoidal, it looks good to me.

:+1:

**Comment by victor-ditay** [comment]
cc @elicoidal

File: mrp_sale_info/README.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+MRP Sale Info
+=============
+
+This module extends the functionality of sale_mrp and adds related fields to Manufacturing Orders and Work Orders:
+
+* Add fields to Manufacturing Orders and Work Orders:
```
Comment (elicoidal) [inline mrp_sale_info/README.rst]: remove line

**Comment by elicoidal** [comment]
LGTM: could be merged quickly

**Comment by victor-ditay** [comment]
cc @elicoidal

**Comment by elicoidal** [comment]
LGTM

**Comment by gurneyalex** [comment]
CLA ok

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**